docs/contributing.rst

```diff
@@ -8,8 +8,8 @@ If you're interested in getting involved in the development of Modin, but aren't
 where start, take a look at the issues tagged `Good first issue`_ or Documentation_.
 These are issues that would be good for getting familiar with the codebase and better
 understanding some of the more complex components of the architecture. There is
-documentation here about the architecture_ that you will want to review in order to get
-started.
+documentation here about the :doc:`architecture </developer/architecture>` that you will
```

Comment:
I suppose this link should lead to `architecture.rst` opened but that doesn't happen. Please, check all the changed references.

Reply:
No, this links work only as a separate site (https://modin.readthedocs.io/en/latest/). Sphinx dosn't provide browsing by internal links via github.